### ISSUE TITLE
Highlight camera

### ISSUE THREAD

**Issue by sqrw** [sqrw/sqrw.github.io#34]
I think this is what missile cam (as much as I hate it) is

**Comment by StarCitizenTracker** [comment]
Highlight camera is supposed to show cool deployments, not missile chases. It should be toggle-able which should be easy to find a screenshot of the menu option.

> "what we call a highlight camera system so like for like **cool deployments and stuff like your missile Bay's open a gun turrets deploy it will like cut out for for like a brief moment** so to show the deployment and then go back into your first-person mode in the car that's pretty cool **you can turn it off** if you want "

**Comment by sqrw** [comment]
Good point, I read it as 'deploying the missile' which is part of missile cam, I think it's somewhere in between but probably not close enough to failry say it's in alpha
